Makefile

```diff
@@ -9,14 +9,18 @@ SIGN_KEY          ?= ${HOME}/.minisign/go-camo.key
 # app specific info
 APP_NAME          := go-camo
 APP_VER           := $(shell git describe --always --tags|sed 's/^v//')
+BRANCH_VAR        := github.com/prometheus/common/version.Branch
```

Comment:
What is the goal of these additions (here and in the _flags variable)?

Reply:
ah. I see. Adding version info the metrics.
I don't think I want to do that. Folks should know what version they are deploying. I don't see how adding this is especially useful. It may also leak unexpected information if folks aren't careful.

Reply:
There's a difference between what you want deployed, and what is running. Hence the need to monitor it.

Reply:
I also don't see a huge information leak with the version info, users generally protect their metrics endpoints anyway. Exposing build info is extremely common across the Prometheus ecosystem. Which is why there's a handy wrapper in prometheus/common. :smiley:

Reply:
If we do add it, we will probably want to call it out in the readme at least, that version info will be exposed.

Reply:
Sure, I'll update the docs.

Reply:
Ok, I updated the README with info on the metrics.